src/rulesets/LinearAlgebra/factorization.jl

```diff
@@ -7,34 +7,34 @@ using LinearAlgebra.BLAS: gemv, gemv!, gemm!, trsm!, axpy!, ger!
 
 function rrule(::typeof(svd), X::AbstractMatrix{<:Real})
     F = svd(X)
-    function svd_pullback(Ȳ::NamedTuple{(:U,:S,:V)})
+    function svd_pullback(Ȳ::Composite{<:SVD})
         ∂X = @thunk(svd_rev(F, Ȳ.U, Ȳ.S, Ȳ.V))
         return (NO_FIELDS, ∂X)
     end
     return F, svd_pullback
 end
 
-function rrule(::typeof(getproperty), F::SVD, x::Symbol)
+function rrule(::typeof(getproperty), F::T, x::Symbol) where T <: SVD
     function getproperty_svd_pullback(Ȳ)
-        if x === :U
-            ∂ = @thunk((; U=Ȳ, S=(zero(F.S)), V=(zero(F.V))))
+        C = Composite{T}
+        ∂F = if x === :U
+            C(U=Ȳ,)
         elseif x === :S
-            ∂ = @thunk((; U=(zero(F.U)), S=Ȳ, V=(zero(F.V))))
+            C(S=Ȳ,)
         elseif x === :V
-            ∂ = @thunk((; U=(zero(F.U)), S=(zero(F.S)), V=Ȳ))
+            C(V=Ȳ,)
         elseif x === :Vt
             # TODO: This could be made to work, but it'd be a pain
```

Comment:
```suggestion
            # TODO: This could be made to work, but it'd be a pain... or would it?
```

Reply:
can we just delete this comment and leave just the reference to the issue?

Reply:
can we just delete this comment and leave just the reference to the issue?